plugins/gimp/file-jxl-load.cc

```diff
@@ -85,10 +105,16 @@ bool LoadJpegXlImage(const gchar *const filename, gint32 *const image_id) {
   // grand decode loop...
   JxlDecoderSetInput(dec.get(), compressed.data(), compressed.size());
 
+  if (JXL_DEC_SUCCESS != JxlDecoderSetProgressiveDetail(
```

Comment:
Thanks for the ping @mo271. Unfortunately, discussion in Discord led me to realize that this patch wasn't *quite* perfect. Better output was possible, and will become more possible when API improves: I'll open an issue for that soon. But, I made a small change here which I forgot to push upstream.

For the moon image it results in _ever so slightly better_ output:

![image](https://user-images.githubusercontent.com/838783/201637096-71709396-af99-4ce4-9a2c-4ece7c2e037c.png)

Not talking about the color correction, talking about this tiny segment!

![image](https://user-images.githubusercontent.com/838783/201637159-93164854-57ea-4c19-b0d2-3ccc59d0613d.png)

Reply:
LGTM